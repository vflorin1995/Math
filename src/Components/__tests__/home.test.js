import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import Home from '../home';

describe('Jest Snapshot testing suite', () => {
  it('Matches DOM Snapshot', () => {
    const domTree = renderer.create(<Router><Home /></Router>).toJSON();
    expect(domTree).toMatchSnapshot();
  });
});
