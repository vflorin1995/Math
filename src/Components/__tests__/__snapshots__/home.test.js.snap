// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite Matches DOM Snapshot 1`] = `
<div>
  <h2>
    Welcome to our page
  </h2>
  <p>
    An electronic calculator is typically a portable electronic device used to perform calculations, ranging from basic arithmetic to complex mathematics. The first solid-state electronic calculator was created in the early 1960s. Pocket-sized devices became available in the 1970s, especially after the Intel 4004, the first microprocessor, was developed by Intel for the Japanese calculator company Busicom. Modern electronic calculators vary from cheap, give-away, credit-card-sized models to sturdy desktop models with built-in printers. They became popular in the mid-1970s was the incorporation of integrated circuits reduced their size and cost. By the end of that decade, prices had dropped to the point where a basic calculator was affordable to most and they became common in schools. Computer operating systems as far back as early Unix have included interactive calculator programs such as dc and hoc, and calculator functions are included in almost all personal digital assistant type devices, the exceptions being a few dedicated address book and dictionary devices. In addition to general purpose calculators, there are those designed for specific markets. For example, there are scientific calculators which include trigonometric and statistical calculations. Some calculators even have the ability to do computer algebra. Graphing calculators can be used to graph functions defined on the real line, or higher-dimensional Euclidean space. As of 2016, basic calculators cost little, but scientific and graphing models tend to cost more. With the very wide availability of smartphones, tablet computers and personal computers, dedicated hardware calculators, while still widely used, are less common than they once were. In 1986, calculators still represented an estimated 41% of the worlds general-purpose hardware capacity to compute information. By 2007, this had diminished to less than 0.05%
  </p>
</div>
`;
